src/frontend/Komponenter/Oppgavebenk/OppgavebenkApp.tsx

```diff
@@ -28,7 +34,14 @@ const TabellContainer = styled.div`
 
 export const OppgavebenkApp: React.FC = () => {
     const { axiosRequest, erSaksbehandler } = useApp();
-    const [oppgaveRessurs, settOppgaveRessurs] = useState<OppgaveRessurs>(byggTomRessurs());
+    const [oppgaveRessurs, settOppgaveRessurs] = useState<Ressurs<IOppgaverResponse>>(
+        byggTomRessurs()
+    );
+
+    const [oppgaveRespons, settOppgaveRespons] = useState<IOppgaverResponse>({
+        antallTreffTotalt: 0,
+        oppgaver: [],
+    });
```

Comment:
Vi var litt usikre på dette her. Vi lagde en ny state `oppgaveRespons` for å enklere kunne oppdatere en enkelt oppgave i oppgavelisten/tabellen. Men vi vil fortsatt ta vare på ressursen for å kunne bruke `DataViewer` og enkelt håndtere laster/feil osv. Derfor har vi duplisert staten, som heller ikke virker helt riktig. Innspill tas i mot med takk :)

Reply:
Ser den! Hadde kanskje vurdert å flytte ned oppgaveliste-statet og oppdaterOppgave en nivå ned, dvs tl OppgaveTabell. Blir litt 
```
const OppgaveTabell: React.FC...
const [oppgaver, settOppgaver] = useState<IOppgaverResponse>(oppgaveResponse);
    useEffectNotInitialRender(() => {
        settOppgaver(oppgaveResponse)  
    }, [oppgaveResponse]);
    
    const oppdaterOppgave = 
```

Reply:
Takk, vi forsøkte oss på det. Er jo fortsatt et lite antipattern å duplisere props inn som state, men tenker vi kan leve med det :)